packages/embeds/embed-core/src/embed.ts

```diff
@@ -1,11 +1,20 @@
+/// <reference types="../env" />
 import { FloatingButton } from "./FloatingButton/FloatingButton";
 import { Inline } from "./Inline/inline";
 import { ModalBox } from "./ModalBox/ModalBox";
-import type { methods, UiConfig } from "./embed-iframe";
+import type { InterfaceWithParent, interfaceWithParent, UiConfig } from "./embed-iframe";
 import css from "./embed.css";
+import type { EventData, EventDataMap } from "./sdk-action-manager";
 import { SdkActionManager } from "./sdk-action-manager";
-import allCss from "./tailwind.generated.css";
-
+import allCss from "./tailwind.generated.css?inline";
```

Comment:
New version of Vite has deprecated importing text like this without ?inline parameter.